scripts/contract/contract_check.py

```
@@ -38,7 +38,7 @@
     raise
 
 POLICY_REPO_API = "repos/Cloudbird-Software/.github/contents/governance/policy/contracts.yaml"
-ADR_DIR_API = "repos/Cloudbird-Software/agent-registry/contents/decisions"
+ADR_DIR_API = "repos/Cloudbird-Software/archive/contents/adr"  # ADR 家园单仓化（ADR-0085）：agent-registry/decisions 已退役停维
```

Ruff (0.16.2)

[warning] Comment contains ambiguous `（` (FULLWIDTH LEFT PARENTHESIS). Did you mean `(` (LEFT PARENTHESIS)?

(RUF003)